.circleci/build_prod_config.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import json
+
+with open('../dist/js-hash.json', 'r') as js_hash_file, open('../dist/css-hash.json', 'r') as css_hash_file:
+    js_hash = json.load(js_hash_file)
+    css_hash = json.load(css_hash_file)
+
+    # Bah, mutations
+    full_hash = js_hash.copy()
+    full_hash.update(css_hash)
+
+    prodConfigJson = (json.dumps({'hashedAssets': full_hash}))
+    prodConfig = open('../server/config/prod.json', 'w')
+    prodConfig.write(prodConfigJson)
+    prodConfig.close()
```

Comment:
This `f = open(...); ...; f.close()` pattern is generally no longer used – what if you hit an error while the file is still open, and you don’t close it properly? Better is to use a *context manager*, like so:

```python
with open('../server/config/prod.json', 'w') as prodConfig:
    prodConfig.write(prodConfigJson)
```

then even if an exception is thrown in the body, the file still gets closed correctly.